.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+#
+# For syntax help see:
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
+
+
+# The firestore-dpe team is the default owner for anything not
+# explicitly taken by someone else.
+*                               @GoogleCloudPlatform/firestore-dpe
```

Comment:
Is there an `@googleapis/firestore-dpe` that could be used instead of the cross organization (GitHub level) `@GoogleCloudPlatform/firestore-dpe`.

Reply:
There is not. I could make one? Is it normal to duplicate teams across orgs? I'm happy to follow standard practice.

Reply:
Now there is! Updated.